sfm_pc/management/commands/import_google_doc.py

```diff
@@ -274,11 +333,14 @@ def log_error(self, message):
                              message])
 
     def get_confidence(self, confidence_key):
-        key = confidence_key.strip().lower()
-        if key:
-            return CONFIDENCE_MAP[key]
+        if confidence_key in ['1', '2', '3']:
```

Comment:
The new test data uses `1`, `2`, and `3` to refer to confidence scores instead of `Low`, `Medium`, and `High`. This is probably going to be a research standard going forward, so I adjusted the code to reflect that.

Reply:
Lol sadly not ; that was me not paying attention. It should be `Low`, `Medium` or `High`. I've updated the testing data. Sorry :(

Reply:
No worries, it's an easy adjustment. Thanks for catching this!

Reply:
https://github.com/security-force-monitor/sfm-cms/pull/644/commits/f53013364f4525d4421b52d389568649db612bc2 revert support for `1` `2` and `3` as confidence levels.